apps/github-cascading-app/src/cascading/cascading-probot.ts

```diff
@@ -51,19 +51,21 @@ export class CascadingProbot extends Cascading {
       }).catch(() => null)));
 
     const configFileValidResponses = configFileResponses
-      .filter((res) => res && Array.isArray(res.data) && res.data.length > 0);
+      .map((res) => res && !Array.isArray(res.data) && res.data || null)
```

Comment:
Just to be sure I got it right: octokit.repos.getContent returns an array if the path given is a directory, so the goal here is to filter those outs ?

Reply:
yes, we keep only files

Reply:
Should never happen since it's driven by CONFIGURATION_FILES right ?

Reply:
Could happen if someone use named a folder with one of the file names listed :D.
And it is also a way to ensure typing without casting